lib/galaxy/model/mapping.py

```diff
@@ -61,6 +63,7 @@
     Column("deleted", Boolean, index=True, default=False),
     Column("purged", Boolean, index=True, default=False),
     Column("disk_usage", Numeric(15, 0), index=True),
+    Column("deleted_disk_usage", DECIMAL(asdecimal=False), default=0.00, index=True),
```

Comment:
What's the rationale for DECIMAL here?  It may be best to keep it simple and account in the same units as disk_usage.

Reply:
Numeric data type caused page loading problem in `view all histories` (history/view_multiple) when deleted_disk_usage was 0 in my tests. Error message can be like "Decimal('0.00') is not JSON serializable". Changing to `DECIMAL(asdecimal=False), default=0.00` was followed by [https://github.com/pyeve/eve-sqlalchemy/issues/50](https://github.com/pyeve/eve-sqlalchemy/issues/50).

Reply:
@qiagu For consistency, it may be better to update the manager/serializer that generates the responses consumed by history/view_multiple (and other sources) to handle the Numeric correctly, as must have been done for disk_usage?

Reply:
Usually I'd say sure, but since it's a database migration can we nail it down here, to avoid another migration?

Reply:
I see. You're right.